app/controllers/api/v1/medias_controller.rb

```diff
@@ -21,10 +20,19 @@ class MediasController < ApplicationController
       def create
         @path = params[:csv_location]
         account_id = params[:account_id]
-        print("hiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiii")
-         CSV.new(open(@path), :headers => :first_row).each do |row|
-           asset_id = ('0'..'z').to_a.sample(8).join
-           @media = Media.new(
+        access_key_id =  ENV['AWS_ACCESS_KEY_ID'] 
+        secret_access_key = ENV['AWS_SECRET_ACCESS_KEY']
+        bucket_name = ENV['BUCKET_NAME']
+        s3 = Aws::S3::Client.new(region: 'us-east-2', access_key_id: access_key_id, secret_access_key: secret_access_key)
+        resp = s3.get_object({
+        bucket: bucket_name,
+        key: "test.csv"
```

Comment:
our initial design was to get this from params.

Reply:
Yeah. Now I changed the code to get the key directly from the path